addons/binding/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/SomfyTahomaBindingConstants.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.somfytahoma;
+
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
Can you please remove the `@NonNull`? Easiest way should be to add a `@NonNullByDefault` above the class and a `@Nullable` in case something should be allowed to be `null`.

Nullannotations are a nice helper for developers in the IDE but we do not (yet) enforce them. So if you do not want to make use of them you are also free to completely remove them. Just do not add some which we do not use, because they have to be removed at some point then...